shared/src/clients/mail-client.class/interfaces.ts

```diff
@@ -1,6 +1,8 @@
 export interface MailAttachment {
```

Comment:
just thinking out loud: should we refactor this to import [`Attachment`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ba7447ad38c4f69cf4f8d23fc4660fb2e3303700/types/nodemailer/lib/mailer/index.d.ts#L40-L58) and write something like `Pick<Attachment, ...>` to make clear that we are using Nodemailer's API

Reply:
OK have done so in the next commit (easy to undo if not a good idea)